RHIDP-7560 - Install and configure plugins from Extensions page (#1281)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-extensions-enabling.adoc

[id="rhdh-extensions-plugins-enabling_{context}"]
= Enabling plugins installation by using Extensions
Using the extensions UI, the extensions backend now requires additional configuration in the app config, users must explicitly enable the installation and specify the file path from which the plug-in configuration should be read if the file path is missing or invalid installation will be disabled.

[NOTE]
When you visit *Extensions*, before you enable extensions plugin installation, you will see an information message saying that the plugin installation is disabled.

.Prerequisites
* You added a custom Developer Hub application configuration, and have sufficient permissions to modify it.
* You are in development mode.
* You have created the plugin configuration file and added the file path in your `app-config.yaml` file.
* You have the correct RBAC permissions.
// * You have installed the `oc` CLI [TBC].

////  
ISSUES

> I dont't see a warning message when I access RHDH in production mode - Is this the latest version?

Reviewer: ```suggestion
There is no warning message when accessing {product-very-short} in production mode — is this the latest version?
```

Configure RBAC
Add PVC
////

.Procedure
. Switch to *development* mode by editing your app-config.yaml file as follows:
+
.`app-config.yaml` fragment
[source,yaml]
----
auth:
  environment: development # Set to 'development' for development mode
  providers:
    # ... your providers configured for development mode ...
    guest: # Example: Guest provider often has specific settings for dev
      dangerouslyAllowOutsideDevelopment: true # Allows guest login outside of dev mode only if explicitly enabled
----
. Create a plugin configuration file to store plugin configuration data when you enable a plugin by using Extensions.
// dynamic-plugins-test.yaml as configmap

> . To enable extensions plugin installation, you must update your `app-config.yaml` file as follows:

Author: This content has been removed.

+

> .`app-config.yaml` fragment

Author: This content has been removed.
